index.html

```diff
@@ -246,8 +246,8 @@ <h1>
 namespace definition (such as the UUID URN namespace defined in
 [[RFC4122]]). The difference is that a DID Method specification, in
 addition to defining a specific DID scheme, also specifies the
-methods for resolving and deactivating DIDs and writing DID Documents on the
-appropriate <a>DID Registry</a>.
+methods for resolving and deactivating DIDs on and writing DID Documents to
```

Comment:
```suggestion
methods for resolving and deactivating DIDs on, and writing DID Documents to,
```
Commas help comprehension.

A full rewrite of this sentence may be better... perhaps --
```
In addition to defining a specific DID scheme, a DID Method specification 
specifies the methods for resolving and deactivating DIDs on, and writing 
DID Documents to, the appropriate <a>DID Registry</a>.
```